src/screens/Calculator/CalculatorKey/styles.scss

```diff
@@ -0,0 +1,22 @@
+@import './../../../shared/main-styles.scss';
+
+.operator-key {
+  color: white;
+  font-size: 25px;
```

Comment:
For all those keys you have common properties like `font-size`, `color`. You should use some shared class or mixin.

Reply:
Done!